code/modules/reagents/newchem/pyro.dm

```diff
@@ -38,14 +38,14 @@
 /datum/reagent/clf3/on_mob_life(var/mob/living/M as mob)
 	if(!M) M = holder.my_atom
 	M.adjust_fire_stacks(4)
-	M.adjustFireLoss(0.35*M.fire_stacks)
+	M.adjustFireLoss(0.35*1)
```

Comment:
this should defnitely be M.fire_stacks; it does burn damage based on how many fire stacks the mob has.